### ISSUE TITLE
Fine-tuned EasyOCR model with thai_g.pth

### ISSUE THREAD

**Issue by kwankoravich** [JaidedAI/EasyOCR#762]
I'm working on EasyOCR Model and I would like to fine-tune the model. I'm looking into the en_filtered_config.yaml
However, I'm not sure if I would like to fine-tune Thai dataset, how to change 'lang_char' parameter.

The default is lang_char: 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'
I supposed that it should be lang_char: 'กขคฆงจฉชซฌญฎฏฐฑฒณดตถทธนบปผฝพฟภมยรฤลวศษสหฬอฮฯะ ัา ำ ิ ี ึ ื ุ ู ฺเแโใไๆ ็ ่ ้ ๊ ๋ ์ ํ๑๒๓๔๕๖๗๘๙'.

However, I got the error when I load the model. So, could you please suggest to me how to adjust en_filtered_config.yaml?

**Comment by s39674** [comment]
Hi @kwankoravich ! The error probably occurs because the model expects the length of `lang_char` to be **52** while you are inputting a **93** characters string (Although it could be different, I need to see the error ). I suggest to fine tune the Thai dataset, you could always just go back in case something happen.

**Comment by SarmSKunatham** [comment]
Hi @kwankoravich , From what I found in the source code, I use `lang_char` = abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZกขคฆงจฉชซฌญฎฏฐฑฒณดตถทธนบปผฝพฟภมยรลวศษสหฬอฮฤเแโใไะาุูิีืึั่้๊๋็์ำํฺฯๆ and it's work for me in order to do the fine tuning.

**Comment by darwinharianto** [comment]
@SarmSKunatham how do you find that lang_char value inside the source code?

**Comment by darwinharianto** [comment]
yes, you just need to prepare the data and do it based on https://github.com/JaidedAI/EasyOCR/blob/master/trainer/trainer.ipynb

**Comment by khawar-islam** [comment]
@darwinharianto I am confused about the pre-trained weights link. Where I can give the link to the pre-trained weight and where i can download the prestrained weights for Korean, and English recognition? If you check the below link there are no pre-trained weights. 

https://github.com/JaidedAI/EasyOCR/blob/master/trainer/config_files/en_filtered_config.yaml

**Comment by darwinharianto** [comment]
you just need to use the default one
```
import easyocr
easyocr.Reader(['kr']) # is it kr for korea?
```
This will automatically download your pretrained, which is saved at `~/.EasyOCR/model` if you are using ubuntu.
Then change that 
```
saved_model: 'path to the downloaded pretrained(the one inside easyocr's model folder)'
```

**Comment by khawar-islam** [comment]
@darwinharianto yes, I know but this KR model is not robust for Korean handwritten recognition. Therefore, I want to fine-tune the KR model on KR handwritten samples (2M)

**Comment by darwinharianto** [comment]
I think I don't understand what are you trying to achieve. 
When I try to fine tune a model, I load the previous model, then run training using my custom dataset on the model. The resulting model would be my fine tuned model.

In your case, you don't want to use the previous model, but want to fine tune it?

**Comment by khawar-islam** [comment]
@darwinharianto I want to use the previous model and then I will do the fine-tuning. In simple words, I want to do the same as your view "When I try to fine tune a model, I load the previous model, then run training using my custom dataset on the model. The resulting model would be my fine tuned model."

**Comment by khawar-islam** [comment]
Yes, I find the EasyOCR model but where I can give a model link for fine-tuning?
